storage/grpc_metrics_test.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	"go.opentelemetry.io/otel/attribute"
+	sdkmetric "go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/metric/metricdata"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"google.golang.org/api/iterator"
+	estats "google.golang.org/grpc/experimental/stats"
+)
+
+func TestMetrics(t *testing.T) {
+	ctx := context.Background()
+	grpcClient, err := NewGRPCClient(ctx)
```

Comment:
Does this work without auth? If not we should probably make it an integration test. Or try `option.WithoutAuthentication`

Reply:
Requires auth; I need to update to not run with short tests.

Reply:
If it's an integration test, move to the file with the other integration tests.